src/reducers/sources.js

```diff
@@ -136,35 +136,55 @@ function _updateText(state, action : any) : Record<SourcesState> {
   }));
 }
 
-function removeSourceFromTabList(tabs, id) {
-  return tabs.filter(tab => tab.get("id") != id);
+function readTabs() {
+  if (typeof localStorage === "undefined") {
+    return;
+  }
+
+  return JSON.parse(localStorage.getItem("tabs"));
```

Comment:
Hey @arthur801031, I just realized we can replace this localStorage stuff with **PREFS**

Here's a link for how to setup a new pref
https://github.com/devtools-html/debugger.html/blob/master/docs/local-development.md#prefs